src/lib/containers/HasAccess.tsx

```diff
@@ -243,18 +274,50 @@ export default class HasAccess extends React.Component<
     }
   }
 
+  refresh = () => {
+    this.getRestrictionInformation()
+    this.getFileEntityFileHandle()
+  }
+
+  handleGetAccess = () => {
+    const { token, entityId } = this.props
+    const { displayAccessRequirement } = this.state
+    SynapseClient.getAllAccessRequirements(token, entityId).then(
+      requirements => {
+        if (checkUnSupportedRequirement(requirements)) {
+          window.open(
+            `${getEndpoint(
```

Comment:
1. Comment out 287 -> 300
2. Call window.open with prop logic for forceSamePage - window.open('synapse.org....', forceSamePage ? '_self': '_blank')
3.